app/controllers/HasCountryOfOriginChangeController.scala

```diff
@@ -52,6 +52,13 @@ class HasCountryOfOriginChangeController @Inject() (
 
   def onPageLoad(mode: Mode, recordId: String): Action[AnyContent] =
     (identify andThen profileAuth andThen getData andThen requireData) { implicit request =>
+      auditService
```

Comment:
we need to remove auditing logic from CountryOfOriginController otherwise it will audit log twice